isthmus/src/test/java/io/substrait/isthmus/OneWaySimpleTest.java

```diff
@@ -0,0 +1,36 @@
+package io.substrait.isthmus;
+
+import com.google.common.collect.Lists;
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.google.protobuf.util.JsonFormat;
+import io.substrait.proto.Plan;
+import org.apache.calcite.sql.parser.SqlParseException;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.List;
+
+public class OneWaySimpleTest extends PlanTestBase {
+  @Test
+  public void windowFunctionsSql2Sub() throws IOException, SqlParseException {
+    SqlToSubstrait sqlToSubstrait = new SqlToSubstrait();
+    List<String> ddlStmts = Lists.newArrayList(asString("tpch/schema.sql"));
+    print(
```

Comment:
The test did not verify the output? 

Also, will it be good to add more example of window function, to cover more usage cases?

Reply:
benefit line 119, for the invocator to set or reset the window expression's related relnode and type

Reply:
wait for the substrait to merge the window function PR, I think this test case it temp, after the reverse translation done, will be refactored to the roundtrip test case.

Reply:
The one test you have is window aggregate function? would it make sense to add at least a window non-aggregate function, such as lag(), rank(), etc? This is to exercise the new code you added.

Reply:
yes

wait for this [substrait PR](https://github.com/substrait-io/substrait/pull/224) to be merged,but I tested those window functions at local. Maybe I could add those test case at this PR,but would have a bad pr experience like this [one](https://github.com/substrait-io/substrait-java/pull/38).